internal/cni-server/server.go

```diff
@@ -43,7 +43,7 @@ type server struct {
 // the path this the unix path to listen.
 func NewServer(unixSockPath string, bpfMountPath string) Server {
 	if unixSockPath == "" {
-		unixSockPath = "/var/run/merbridge-cni.sock"
+		unixSockPath = "/host/var/run/merbridge-cni.sock"
```

Comment:
cni plugin is running on host, not in container.
so, it should be `/var/run/merbridge-cni.sock`.